packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -818,7 +817,6 @@ export default function useAutocomplete(props) {
         // Prevent blur
         event.preventDefault();
       },
-      ...ListboxProps,
```

Comment:
@oliviertassinari if we are not passing ListboxProps via getListboxProps then we need to pass them down directly in Autocomplete.js


```jsx
<ListboxComponent className={classes.listbox} {...getListboxProps()}>
```

Reply:
Yes, you are right, thank you for double checking, we need to solve this

Reply:
So do you want to add it to back getListboxProps or pass directly like below? 

```jsx
<ListboxComponent className={classes.listbox} {...getListboxProps()} {...ListboxProps }>
```

Reply:
I think that the hook API shouldn't be involved in the logic. The code snippet you have shared looks great :).